.gitlab-ci.yml

```diff
@@ -142,7 +136,6 @@ Debian10:
     distro: debian:10
     playbook: test.yml
   <<: *testing_script
-  allow_failure: true
```

Comment:
why is this line removed? Is it related to the redis bug? If not I would leave it there and address it later if needed